spec/e2e/association_proxy_spec.rb

```diff
@@ -31,6 +31,30 @@
       has_many :in, :lessons, model_class: :Lesson, origin: :exams_given
       has_many :out, :students, type: :has_student, model_class: :Student
     end
+
+    stub_active_node_class('Person') do
+      property :name
+
+      has_many :out, :knows, model_class: 'Person', type: nil
+      has_many :in, :knows_me, origin: :knows, model_class: 'Person'
```

Comment:
why create an unused association in specs? Specs should have only the minimum necessary setup.
Even if used a better name would be e.g. `known_by`.